src/app/main/table/table-routing.module.ts

```diff
@@ -22,21 +22,21 @@ export const routes: Routes = [
   {
     path: '', component: TableComponent, children: [
       { path: '', redirectTo: 'basic', pathMatch: 'prefix' },
-      { path: 'basic', component: TableBasicComponent },
-      { path: 'fixed', component: TableFixedComponent },
-      { path: 'renderer', component: TableRendererComponent },
-      { path: 'custrenderer', component: TableCustomRendererComponent },
-      { path: 'editor', component: TableEditorComponent },
-      { path: 'total', component: TableTotalComponent },
-      { path: 'calculatedcolumn', component: TableCalculatedColumnComponent },
-      { path: 'pagination', component: TablePaginationComponent },
-      { path: 'contextmenu', component: TableContextMenuComponent },
-      { path: 'horizontalscroll', component: TableHorizontalScrollComponent },
-      { path: 'insertablerow', component: TableInsertableRowComponent },
-      { path: 'align', component: TableAlignmentComponent },
-      { path: 'multipleSort', component: TableMultipleSortComponent },
-      { path: 'rowexpandable', component: TableRowExpandableComponent },
-      { path: 'rowgroup', component: TableRowGroupComponent }
+      { path: 'basic', data: { oAppHeaderTitle: 'Basic table' }, component: TableBasicComponent },
+      { path: 'fixed', data: { oAppHeaderTitle: 'Fixed header or footer table' }, component: TableFixedComponent },
+      { path: 'renderer', data: { oAppHeaderTitle: 'Renderer table' }, component: TableRendererComponent },
```

Comment:
Maybe a more specific/descriptive title would fit better. Something like "Table with cell renderers"

Other header titles In which I would be more specific:
'Custom renderer table' => 'Table with custom cell renderers'
'Table editor' => 'Table with cell editors'
'Table totals' => 'Table with total row'